### PR TITLE
Rename occurrences of was-liberty to websphere

### PR DESCRIPTION
**Pull request by fritze2**

### CODE REVIEW

**Comment by fritze2** [comment]
#build

File: dev/build.changeDetector/src/com/ibm/ws/infra/depchain/ChangeDetector.java [hunk 1]
```diff
@@ -424,12 +424,12 @@ private static void processOverallFeatureDeps(JsonObject fatMap, Map<String, Set
...
         // Convert: https://github.com/OpenLiberty/open-liberty/pull/5333
         // to:      https://github.com/OpenLiberty/open-liberty/pull/4947.diff
-        // to:      https://github.ibm.com/api/v3/repos/was-liberty/WS-CD-Open/pulls/13986/files
+        // to:      https://github.ibm.com/api/v3/repos/websphere/WS-CD-Open/pulls/13986/files
         String prNumber = prURL.substring(prURL.lastIndexOf("/"));
 
         // Decide which GitHub we need to make an API call to and choose the appropriate URL/access key
         String apiURL = prURL.contains("github.ibm") ? //
-                        "https://github.ibm.com/api/v3/repos/was-liberty/WS-CD-Open/pulls" + prNumber + "/files" : //
+                        "https://github.ibm.com/api/v3/repos/websphere/WS-CD-Open/pulls" + prNumber + "/files" : //
```
Comment (jhanders34) [inline dev/build.changeDetector/src/com/ibm/ws/infra/depchain/ChangeDetector.java:432]: Do we even need this logic?  Isn't it always open liberty here?  The GHE repo has its own copy of this code.

File: dev/cnf/ant/liberty-ghe.xml [hunk 2]
```diff
@@ -135,18 +135,18 @@ ${commits.since.latest.green.build}
     </target>
     
     <target name="taggingChecks">
```
Comment (jhanders34) [inline dev/cnf/ant/liberty-ghe.xml:137]: Is this code even used with open liberty.  Can we delete the task complete?  Or maybe even delete the entire file?

File: dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/CommonTests.java [hunk 3]
```diff
@@ -699,7 +699,7 @@ private static void setupSSLClient() {
 
         /*
```
Comment (jhanders34) [inline dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/CommonTests.java:700]: Maybe this commented out code should just be removed?

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/cnf/ant/liberty-ghe.xml (hunk 2 above)
Comment (fritze2) [reply dev/cnf/ant/liberty-ghe.xml]: I'll try removing the cnf/ant directory in the next build.

File: dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/CommonTests.java (hunk 3 above)
Comment (fritze2) [reply dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/CommonTests.java]: Removed.

File: dev/build.changeDetector/src/com/ibm/ws/infra/depchain/ChangeDetector.java (hunk 1 above)
Comment (fritze2) [reply dev/build.changeDetector/src/com/ibm/ws/infra/depchain/ChangeDetector.java]: Changed to only set the open-liberty apiURL.

**Comment by fritze2** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by fritze2** [comment]
#run-libby-bot

**Pull request merged**